src/service/router/test_router.py

```
@@ -0,0 +1,35 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+"""
+
+import os
+import unittest
+
+os.environ['OSMO_POSTGRES_PASSWORD'] = 'test'
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "OSMO_POSTGRES_PASSWORD"

(S105)